tests/test_tools/test_convert_dgv.py

```diff
@@ -0,0 +1,20 @@
+import os
+import pandas as pd
+import pytest
+from tools.convert_dgv import (
+    convert_to_dictionary,
+)
+
+
+@pytest.mark.parametrize(
+    'filename,expected_file',
+    [
+        ['dgv_test.tab', 'dgv_test_expected.tab'],
+    ],
+)
+def test_dgv_examples(filename, expected_file):
```

Comment:
rather than test just the convert function you should test the main function and mock sys.argv so that we have it end to end. then test that the file you output (pass a temp path) matches what you expect.

Reply:
Addressed in [473afbc](https://github.com/bcgsc/mavis/pull/336/commits/473afbc04ece54e5d4fdb5ebdea6b4b7df4d9d40)

Reply:
wrong commit?

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).